xcp_d/utils/plot.py

```diff
@@ -342,51 +342,56 @@ def confoundplotx(time_series,
             maximum_value.append(max(time_series[c]))
             minimum_value.append(min(time_series[c]))
 
-            # Threshold fd at 0.1, 0.2 and 0.5 and plot
-            time_series_axis.axhline(y=1, color='lightgray',
-                                     linestyle='-', linewidth=10, alpha=0.5)
+            # Threshold fd at 0.05, 0.1, 0.2, 0.5, 1.0 mm and plot
+            time_series_axis.axhline(y=0.05, color='gray', linestyle='-', linewidth=5)
```

Comment:
@kahinimehta you modified a bunch of these parameters in #530. Would you mind taking a look at the changes here and weighing in?